var/spack/repos/builtin/packages/mpip/package.py

```diff
@@ -59,9 +59,8 @@ class Mpip(AutotoolsPackage):
     depends_on('python@:2', when='@3.4.1', type='build')
     depends_on('mpi')
 
-    #  Ideally would use libunwind, but provide backtrace and
-    #    setjmp functionality, if needed
-    #  depends_on('unwind')
+    #  '+setjmp' adds '--disable-libunwind' to the confiure args
+    depends_on('libunwind', when='@3.5: +libunwind ~setjmp')
```

Comment:
I think this should read:
```suggestion
    depends_on('unwind', when='@3.5: +libunwind ~setjmp')
```
if the package can possibly make use of other providers.

Reply:
Thanks @alalazo, however, there is no package for `unwind`, I think the dependency should be `libunwind`. Maybe I'm missing something?

Reply:
@lukebroskop Unwind is a virtual dependency, like `mpi` or `lapack` or `java`. If you give this command:
```console
$ spack providers unwind
unwind:
apple-libunwind  libunwind
```
you can check which packages can be swapped in for `unwind`.

Reply:
Thanks @alalazo! I didn't realize `unwind` was a virtual dependency.